### DISCUSSION TITLE
What is the reason for the decrease in the proportion of accurate points in the NVT complex

### DISCUSSION THREAD

**Discussion by LJyanyan** [deepmodeling/dpgen#1267] [Q&A]
**What is the reason for the rise and fall of the proportion of accurate points when running NVT ensemble? I tested the max_devi_f in the next model_devi.out file, and they are all in the upper and lower limits of the maximum force deviation I set, and I have analyzed the distribution chart of the upper and lower limits of the maximum force deviation and the maximum force deviation, and I feel that there should be no problem with the settings, and the lammps settings are the same.**
Here is my lammps related setup.
![image](https://github.com/deepmodeling/dpgen/assets/109069344/78203e04-3517-48fc-aec4-830d52961dd7)
Structure of the BULK system 14-17 rounds of accurate point ratio.
![image](https://github.com/deepmodeling/dpgen/assets/109069344/01b15c7b-5111-4125-825e-ed662909e48e)
Structure of the surf system 14-17 rounds of accurate point ratio.
![image](https://github.com/deepmodeling/dpgen/assets/109069344/0dd53676-0e4e-4bd0-a32f-a971758585dc)
I test the maximum force deviation and force deviation distribution plot.Round 8 of these is the result of complete convergence in the NPT ensemble.
![1688546815134](https://github.com/deepmodeling/dpgen/assets/109069344/e082d01c-a921-4c43-ac0d-664716a69b38)
![image](https://github.com/deepmodeling/dpgen/assets/109069344/e77031e6-4419-4b0a-906c-50f65b284171)
**Later, I tried to modify the gradient of the temperature range, from the original 200, it will be 100, and in the second round, there was still a decrease in the proportion of accurate points.**
**Later, I tried to enlarge trust low, the original trust low=0.05, changed to trust low=0.07, and the proportion of accurate points still decreased when running the second round.**
![21](https://github.com/deepmodeling/dpgen/assets/109069344/65dd11b9-f0d1-4e20-b532-306e0af9a084)
![20](https://github.com/deepmodeling/dpgen/assets/109069344/90243048-2ded-4568-9c04-941dde88e9d3)
**I have also tried to test common data sets, for example, I tested the single point generated in round 19 with the potential function of round 20 and the potential function of round 21. There is a little increase in the energy RMSE and the force decreases, and the result is as follows, the left is the potential function test result of 21 rounds, and the right is the potential function of 20 rounds.**
![微信图片_20230705171047](https://github.com/deepmodeling/dpgen/assets/109069344/f480adaf-cdf2-4af9-be02-4663b9e51160)
**I have also tested those single points where the proportion of accurate points decreases, and made the energy and force scatter plots of DP and DFT as follows, and I feel that the accuracy is acceptable.**
![DPF_DP_e_17](https://github.com/deepmodeling/dpgen/assets/109069344/3864f316-885d-4348-bb2c-05dfdff0936f)
![DFT_DP_f_17](https://github.com/deepmodeling/dpgen/assets/109069344/c4999cd7-672d-434d-8010-835c9cc1d653)
I don't know what's wrong with my model, but I would appreciate it if you could help me with some **suggestions**

**Comment by njzjz** [comment]
There are two aspects:

1. In each iteration, different models and seeds are used to generate different trajectories. It is unfair to compare the model deviation of models in different trajectories. If you want to confirm there is nothing wrong, I suggest comparing the model deviation using the same trajectory or the same data.

2. In each iteration, do you start the training process from scratch, or continue from the last iteration? The latter will improve the accuracy.

**Reply by LJyanyan** [reply]
Thank you for your suggestion, I used the 19th round of data set to do the verification with the potential function of the 20th round and the potential function of the 21st round, I don't know if it is reasonable? I used the method of starting with. 
I want to know what causes the trend of the proportion of iterative accurate points to rise and fall, and what errors will I troubleshoot that are reasonable?
